changelog/private.dd

```diff
@@ -0,0 +1,3 @@
+Deprecate the use of private selectively imported members
+
+$(LINK2 $(ROOT_DIR)spec/attribute.html#visibility_attributes, The specification states) that a private member is visible only from within the same module. Prior to this release, due to a bug, some imported private members were visible from other modules, violating the specification. Beginning with this release, accessing said private members from outside the module in which they are declared will result in a deprecation message which has been scheduled to be changed to an error in approximately 1 year.
```

Comment:
`s/some imported private members/private selective imports/`
Also just mention that it has been deprecated, we have a general schedule for deprecations (6 months), but deviate from that when it seems too disturbing.